src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2ConnectionTests.cs

```diff
@@ -428,8 +428,7 @@ public async Task StreamPool_MultipleStreamsInSequence_PooledStreamReused()
             await StopConnectionAsync(expectedLastStreamId: 3, ignoreNonGoAwayFrames: false);
         }
 
-        [Fact]
-        [QuarantinedTest]
+        [Fact(Skip = "Causes tests to not finish. See https://dev.azure.com/dnceng/internal/_build/results?buildId=580307&view=logs")]
```

Comment:
While we're at it, can we add `.DefaultTimeout()` to the `await output._dataWriteProcessingTask;` in the test. That looks like to most suspicious line to me.